service/apis/entity_service_test.go

```diff
@@ -13,8 +13,8 @@ import (
 func TestCreateEntity(t *testing.T) {
 	dc := db.NewDGClient("127.0.0.1:9080")
 	defer dc.Close()
-	ms := NewMetaService(dc)
-	s := NewEntityService(dc, ms)
+	//	ms := NewMetaService(dc)
```

Comment:
can we delete this line instead of comment out?

Reply:
will remove those commented lines